TYPO3.Neos.NodeTypes/Resources/Private/TypoScript/Root.ts2

```diff
@@ -63,9 +63,11 @@ prototype(TYPO3.Neos.NodeTypes:MultiColumn) < prototype(TYPO3.Neos:Content) {
 	layout = ${q(node).property('layout')}
 	attributes.class = ${'container columns-' + q(node).property('layout')}
 	columns = TYPO3.TypoScript:Collection {
+		@context.columnLayout = ${q(node).property('layout')}
```

Comment:
I wanted to scope the `layout` context variable as much as possible so I put it in there. As there is no speed difference in using `this.layout` and just repeating the FlowQuery I choose the scoping.
I figured that `layout` may be used as a property name in the single columns and so wanted to prefix this information inside the context to avoid confusion, also just having a variable layout in the context tells you nothing about where it comes from.